core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java

```diff
@@ -285,14 +285,15 @@ private boolean isBtcOutputOfBurnFeeTx(TempTxOutput tempTxOutput) {
                         // We always have the BSQ change before the burned BSQ output if both are present.
                         checkArgument(optionalOpReturnIndex.isPresent());
                         if (index != optionalOpReturnIndex.get() - 1) {
-                            return false;
+                            break;
```

Comment:
I don't see any problem with that change but as it is a pure "cosmetic" change I prefer to not touch the code. Some could argue the early return is more readable. I don't have a strong opinion and the break is more consistent but the evaluating benefit / risk just does not justify the change IMO even it seems a trivial and very low risk change.